django/db/models/functions.py

```diff
@@ -137,3 +137,16 @@ class Upper(Func):
 
     def __init__(self, expression, **extra):
         super(Upper, self).__init__(expression, **extra)
+
+
+class Now(Func):
+    template = 'CURRENT_TIMESTAMP'
+
+    def __init__(self, output_field=None, **extra):
+        if output_field is None:
+            output_field = DateTimeField()
+        super(Now, self).__init__(output_field=output_field, **extra)
+
+    def as_postgresql(self, compiler, connection):
+        self.template = 'STATEMENT_TIMESTAMP()'
```

Comment:
Could you add a comment with your explanation about why CURRENT_TIMESTAMP is inappropriate on PostgreSQL.